### PR TITLE
Add ImageNet trained VGG models and fix weight initialization

### PR DESCRIPTION
**Pull request by colesbury**

### CODE REVIEW

**Comment by Maratyszcza** [comment]
Which batch size/learning rate/schedule/weight decay were used in training?

**Comment by colesbury** [comment]
LR=0.01 with default schedule (divide by 10 every 30 epochs). Everything
else was the default settings (batch-size=256, weight-decay=1e-4)

**Comment by fmassa** [comment]
We should probably keep a table tracking final validation performances and the hyper-parameters used during training, and add it to the README.

**Comment by colesbury** [comment]
Yeah, that's a good idea.
